nautobot/circuits/tables.py

```diff
@@ -9,7 +9,24 @@
     TagColumn,
     ToggleColumn,
 )
-from .models import Circuit, CircuitType, Provider
+from .models import Circuit, CircuitType, Provider, ProviderNetwork
+
+
+#
+# Provider Network
+#
+
+
+class ProviderNetworkTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True)
+    provider = tables.Column(linkify=True)
+    tags = TagColumn(url_name="circuits:providernetwork_list")
+
+    class Meta(BaseTable.Meta):
+        model = ProviderNetwork
+        fields = ("pk", "name", "provider", "description", "tags")
```

Comment:
I am not sure about this, because comments can contain Markdown. I think, this should not be shown within a table. `description` can be shown and this is only text.

Reply:
Also, it is not done anywhere else.

Reply:
Good call, thanks!